app/views/users/new.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Join Chitter today.</h1>
+
+<form action='/users' method='post'>
+	<p>
+		<input type='text' placeholder='Full name' name='full_name' value='<%= @user.full_name %>' required>
+	</p>
+	<p>
+		<input type='email' placeholder='Email' name='email' value='<%= @user.email %>' required>	
+	</p>
+	<p>
+		<input type='password' placeholder='Password' name='password' required>	
+	</p>
+	<p>
+		<input type='password' placeholder='Password confirmation' name='password_confirmation' required>	
+	</p>
+		<input type='submit' value='Sign up'>
+</form>
```

Comment:
I found that  I can't change an indentation on github  if firstly it was uoloaded with wrong indentation, even if I correct them lately.